lib/import-export-visitor.js

```diff
@@ -184,8 +184,8 @@ class ImportExportVisitor extends Visitor {
   visitExportAllDeclaration(path) {
     const decl = path.getValue();
     const hoistedCode =
-      pad(this, "module.importSync(", decl.start, decl.source.start) +
-      getSourceString(this, decl) +
+      pad(this, "module.watch(", decl.start, decl.source.start) +
+      "require(" + getSourceString(this, decl) + ")" +
```

Comment:
Good catch, I think it should go in the one above, since it's usually (always?) only one line, and that line is the first line of the `module.watch` expression.